plugins/modules/ec2_eip.py

```diff
@@ -76,6 +76,24 @@
       - Allocates the new Elastic IP from the provided public IPv4 pool (BYOIP)
         only applies to newly allocated Elastic IPs, isn't validated when O(reuse_existing_ip_allowed=true).
     type: str
+  update_reverse_dns_record:
+    description: Whether to update the reverse DNS record of ec2 Elastic IP Address (eip)
+    required: false
+    type: bool
+  allocation_id:
+    description: The allocation ID of ec2 EIP.
+    required: false
+    type: str
+  domain_name:
+    description: The domain name to modify for the IP address.
+    required: false
+    type: str
+  dry_run:
```

Comment:
This seems new to that collection. I would rather suggest having a global policy on how to deal with `dry_run` for all modules in that collection